packages/web/src/scss/components/Checkbox/README.md

```diff
@@ -26,6 +26,9 @@
 
 See Validation state [dictionary][dictionary-validation].
 
+- To render validation text as a list, use `<ul>` element inside of `<div>`.
+- To render validation text with an icon, add class `Checkbox__validationText--hasIcon` and `<svg>` element inside of `<div>`.
```

Comment:
```suggestion
- To render validation text with an icon, add class `Checkbox__validationText--hasIcon` and `<svg>` element inside of `<span>`.
```

Reply:
We should promote `<span>` element here, because div inside `<label>` is not valid afaik.